playground/mocks/data/idp/idx/authenticator-verification-select-authenticator-ov-m2.json

```diff
@@ -0,0 +1,197 @@
+{
+  "stateHandle": "02im-3M2f6UXHgNfS7Ns7C85EKHzGaKw0u1CC4p9_r",
```

Comment:
Why create a new json and not add to existing select-authenticator .json ?

Reply:
I created a separate response for m2 because I wanted to ensure that these changes still supported the OV response format from devices m1. There are tests for it as well.